### PR TITLE
Add gut microbiome wellness index 2 :hankey:

### PR DESCRIPTION
**Pull request by danielchang2002**
Add GMWI2 recipe. GMWI2 (Gut Microbiome Wellness Index 2) is a robust and biologically interpretable predictor of health status based on gut microbiome taxonomic profiles.

Package code is hosted at this repo: https://github.com/danielchang2002/GMWI2/tree/main

### CODE REVIEW

**Comment by danielchang2002** [comment]
@BiocondaBot please add label

**Comment by danielchang2002** [comment]
@BiocondaBot please fetch artifacts

**Comment by danielchang2002** [comment]
@bioconda/core Hey, would you guys be able to review my PR? Please let me know if anything is wrong with my recipe, thanks!

File: recipes/gmwi2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  version: "{{ version }}"
+
+build:
+  noarch: generic
+  number: 0
+
+source:
+  url: https://github.com/danielchang2002/GMWI2/archive/{{version}}.tar.gz
+  sha256: c7d07cde6d48044234454c7bbd0c8c97b768b0a23d38efa240e9ab29ed3ab87d
+    # path: ..
```
Comment (bgruening) [inline recipes/gmwi2/meta.yaml:15]: can be removed

File: recipes/gmwi2/build.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```
Comment (bgruening) [inline recipes/gmwi2/build.sh:1]: you could move this one -liner into the meta.yaml, see other packages

File: recipes/gmwi2/meta.yaml (hunk 1 above)
Comment (danielchang2002) [reply recipes/gmwi2/meta.yaml:15]: I'm confused, why would it be alright to remove the source? I don't have the python package uploaded to pypi.

File: recipes/gmwi2/build.sh (hunk 2 above)
Comment (danielchang2002) [reply recipes/gmwi2/build.sh]: Got it, I have implemented this, thanks

File: recipes/gmwi2/meta.yaml (hunk 1 above)
Comment (bgruening) [reply recipes/gmwi2/meta.yaml:16]: I was referring to the comment.

Comment (danielchang2002) [reply recipes/gmwi2/meta.yaml]: Ah ok. I've removed the comment.

**Review by bgruening — APPROVED** [review]
Thanks a lot.

**Pull request merged**